cluster-autoscaler/config/crd/provisioning.k8s.io_provisioningrequests.yaml

```diff
@@ -0,0 +1,203 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    api-approved.kubernetes.io: https://github.com/kubernetes/autoscaler/pull/5848
+    controller-gen.kubebuilder.io/version: v0.13.0
+  name: provisioningrequests.provisioning.k8s.io
+spec:
+  group: provisioning.k8s.io
+  names:
+    kind: ProvisioningRequest
+    listKind: ProvisioningRequestList
+    plural: provisioningrequests
+    shortNames:
+    - provreq
+    - provreqs
+    singular: provisioningrequest
+  scope: Namespaced
+  versions:
+  - additionalPrinterColumns:
+    - jsonPath: .metadata.creationTimestamp
+      name: Age
+      type: date
+    name: v1beta1
```

Comment:
This should not start as beta, that carries forward compatibility guarantees for k8s APIs.